stdlib/runtime/numberUtils.gr

```diff
@@ -1018,7 +1058,9 @@ export let itoa64 = (value, radix) => {
       utoa64_dec_core(out + 8n, value, decimals)
     }
   } else if (radix == 16n) {
-    let decimals = WasmI32.shrU(63n - WasmI32.wrapI64(WasmI64.clz(value)), 2n) + 1n + sign
+    let decimals = WasmI32.shrU(63n - WasmI32.wrapI64(WasmI64.clz(value)), 2n) +
+      1n +
+      sign
```

Comment:
Oh man, these infix operators are quite the pain. For example, I'd rather the function call break and then the `+` infix ops would all be on the last line.